Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -276,6 +277,14 @@ YUI.add('ez-fieldeditview', function (Y) {
             this.after('errorStatusChange', this._errorUI);
 
             this._addDOMEventHandlers(_events);
+
+            this.set('isNotTranslatable', this.get('translating') && !this.get('fieldDefinition').isTranslatable);
```

Comment:
`isNotTranslatable` should be marked as read only since it is a computed value so here you should use `_set` and the attribute should have the read only flag